lib/scim/kit/v2/schemas.rb

```diff
@@ -11,6 +11,7 @@ module Schemas
         ENTERPRISE_USER = "#{EXTENSION}:enterprise:2.0:User"
         GROUP = "#{CORE}:Group"
         RESOURCE_TYPE = "#{CORE}:ResourceType"
+        SCHEMA = "#{CORE}:Schema"
```

Comment:
Not sure what the purpose of this is? Couldn't find it specified in RFC 7644 and 7643

Reply:
I found a reference to this in [RFC-7643, Section 7](https://www.rfc-editor.org/rfc/rfc7643.html#section-7).

Reply:
Nice catch. Thanks!!!